Modules/Module_Processing/xASL_init_DataLoading.m

```diff
@@ -47,6 +47,11 @@
 
     
     %% 4. Which data to read
+	% In case we call population module only, then don't attempt to convert rawdata to Legacy, but only read from derivatives
```

Comment:
If we do population module only, then this will only work with data that we already processed in derivatives. Any newly converted BIDS2Legacy will be ignored anyway = hence no point converting them.

A) Yes, but in the future, for now, we still have the option bReadRawdata.
B) That I agree with.

MINOR B) This is not a weird statement, but a necessary statement isequal([0 0 1],[0;0;1]) == 0
MINOR C) I of course read the code. Please read mine and think why I added it this way ;) My code was dealing with special option of missing bReadrawdata and bprocess == [0 0 1] - it wouldn't be possible to seamlessly add that option to the code below.